src/app/models/proptypes.js

```diff
@@ -4,10 +4,10 @@ export const imagePropTypes = {
   model: shape({
```

Comment:
Although CodeClimate is asking for refactoring between this and the proptypes below, I think this will make the file quite difficult to read and therefore will not refactor in this PR.